src/md_lammps.c

```diff
@@ -0,0 +1,507 @@
+#include <md_lammps.h>
+
+#include <md_util.h>
+
+#include <core/md_common.h>
+#include <core/md_str.h>
+#include <core/md_allocator.h>
+#include <core/md_log.h>
+#include <core/md_os.h>
+#include <core/md_array.h>
+#include <core/md_parse.h>
+
+#include <string.h>
+
+bool get_data_format(data_format_t* format, atom_style style) {
+	switch (style)
+	{
+	default:
+		MD_LOG_ERROR("Invalid atom_style");
+		return false;
+	case full:
+		format->ptr = DATA_FORMAT_FULL;
+		format->len = ARRAY_SIZE(DATA_FORMAT_FULL);
+		return true;
+	}
+}
+
+
+float get_mass(md_lammps_atom_mass_t* m, int32_t type, int32_t num_types)
```

Comment:
make this into a static function if it is only accessed from within lammps.c, or is it called elsewhere?

Reply:
Made it static